x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_error.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer, EuiText, EuiButtonEmpty } from '@elastic/eui';
+import React, { memo, useMemo } from 'react';
+import { CrumbInfo, StyledBreadcrumbs } from './panel_content_utilities';
+
+/**
+ * Display an error in the panel when something goes wrong and give the user a way to "retreat" back to a default state.
+ *
+ * @param {function} pushToQueryparams A function to update the hash value in the URL to control panel state
+ * @param {string} errorMessage The message to display in the panel when something goes wrong
+ */
+export const PanelContentError = memo(function ({
+  errorMessage,
+  pushToQueryParams,
+}: {
+  errorMessage: string;
+  pushToQueryParams: (arg0: CrumbInfo) => unknown;
+}) {
+  const crumbs = useMemo(() => {
+    return [
+      {
+        text: i18n.translate('xpack.securitySolution.enpoint.resolver.panel.error.events', {
+          defaultMessage: 'Events',
+        }),
+        onClick: () => {
+          pushToQueryParams({ crumbId: '', crumbEvent: '' });
+        },
+      },
+      {
+        text: i18n.translate('xpack.securitySolution.enpoint.resolver.panel.error.error', {
+          defaultMessage: 'Error',
+        }),
+        onClick: () => {},
+      },
+    ];
+  }, [pushToQueryParams]);
+  return (
+    <>
+      <StyledBreadcrumbs breadcrumbs={crumbs} />
+      <EuiSpacer size="l" />
+      <EuiText textAlign="center">{errorMessage}</EuiText>

```

Comment:
Does this need translation too?

Reply:
It comes in translated already, but you'd have no way of knowing that just looking at the prop name. If I call it `translatedErrorMessage` that should make it more clear, right?